core/services/major_tom/major_tom.py

```diff
@@ -0,0 +1,82 @@
+import asyncio
+from typing import Any, Dict, List
+
+import requests
+from commonwealth.settings.manager import Manager
+from loguru import logger
+
+from session_tracker import SessionTracker
+from settings import SettingsV1
+from system_data import SystemDataGatherer
+
+SERVICE_NAME = "major_tom"
+TELEMETRY_VERSION = 1
+
+
+class MajorTom:
+    """
+    Consolidates data from SessionTracker and SystenDataGatherer in one place and sends it to GroundControl
+    """
+
+    last_session = Dict[str, Any]
+
+    def __init__(self) -> None:
+        self.system_data_gatherer = SystemDataGatherer()
+        self.settings_manager = Manager(SERVICE_NAME, SettingsV1)
+        self.sessions_folder = self.settings_manager.config_folder / "sessions"
+        self.sessions_folder.mkdir(parents=True, exist_ok=True)
+        self.session_tracker = SessionTracker(self.sessions_folder)
+        self.settings = self.settings_manager.settings
+        self.remote = self.settings.remote
+
+    def phoneHome(self, vehicle_data: Dict[str, Any], system_data: Dict[str, Any]) -> int:
+        """
+        Sends data to GroundControl
+        """
+        blacklist = self.settings_manager.settings.blacklist
+        filtered_vehicle_data = {key: value for key, value in vehicle_data.items() if key not in blacklist}
+        filtered_system_data = {key: value for key, value in system_data.items() if key not in blacklist}
+        try:
+            req = requests.post(
+                f"{self.remote}/v1.0/telemetry/send",
+                json={
+                    "version": TELEMETRY_VERSION,
+                    "vehicle_data": filtered_vehicle_data,
```

Comment:
We should use a structure that we will be sure that will not change or will match with ground control, since both need to have the same API, we need to enforce that somehow.